### ISSUE TITLE
Error During Container Instance Creation (Windows)

### ISSUE THREAD

**Issue by davidop** [microsoft/azure-pipelines-ephemeral-agents#8]
![image](https://user-images.githubusercontent.com/7433346/96118845-6553e600-0eec-11eb-95f7-8fb2b0fd6e9f.png)

Error Code: [1]
Error: Couldn't create Azure Container Instance
Script failed with error: The requested resource is not available in the location 'westeurope' at this moment. Please retry with a different resource request or in another location. Resource requested: '2' CPU '8' GB memory 'Windows Server 2019 - After 2B' OS virtual network

**Comment by davidop** [comment]
Any idea?

**Comment by juteasl** [comment]
Did you try a different region besides westeurope to see if would work?

**Comment by davidop** [comment]
Yes, I tried several regions and nothing. If the agent is Linux it works fine but if it is Windows I can't get it to work. Have you managed to create a windows ephemeral agent?

**Comment by mozts2005** [comment]
Container group deployment for a virtual network only supports Linux at this time. 

https://docs.microsoft.com/en-us/azure/container-instances/container-instances-virtual-network-concepts#other-limitations

**Comment by davidop** [comment]
So why do you have the Windows Dockerfile mounted and the drop-down in the step to select the OS?

The concept of ephemeral agents seems very good but it seems that it has remained in a preview version
